test/e2e/events.e2e.js

```diff
@@ -51,4 +51,20 @@ describe('ContactHub', () => {
     });
   });
 
+  describe('getEvents', () => {
+    it('retrieves all Events for a Customer using filters', async () => {
+      const dateFrom = new Date('2017-02-28');
+      const filters = {
+        mode: 'ACTIVE',
+        context: 'WEB',
+        dateFrom
+      };
+      const events = await ch.getEvents(cid, filters);
+      const areWebEvents = events.reduce((acc, v) => acc && v.context === 'WEB', true);
```

Comment:
`reduce` is nice, but [Array.prototype.every()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every) is even shorter! :)